src/components/_tooltip.scss

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2011-2018, Hortonworks Inc. All rights reserved.
+ * Except as expressly permitted in a written agreement between you
+ * or your company and Hortonworks, Inc, any use, reproduction,
+ * modification, redistribution, sharing, lending or other exploitation
+ * of all or any part of the contents of this file is strictly prohibited.
+ */
+
+ .tooltip {
+    .tooltip-inner {
+      padding: 1rem; // if 1rem = 10px
+    }
+    &.tooltip-no-arrow {
+      .arrow {
+        display: none;
+      }
+    }
+    &.tooltip-light {
+      .tooltip-inner {
+        background-color: $white;
+        border: 1px solid darken($white, 14%);
```

Comment:
This color should be specified using a variable. I will be adding the grays as discussed via email and then we can replace this. Same for the next line.